locust/stats.py

```diff
@@ -784,25 +784,47 @@ def on_worker_report(client_id: str, data: Dict[str, Any]) -> None:
 
 
 def print_stats(stats: RequestStats, current=True) -> None:
+    stat_summary = get_stats_summary(stats, current)
+    for summary in stat_summary:
```

Comment:
skip intermediate "start_summary" variable. Also, maybe "line" is a more appropriate name than "summary"?